experimental/veggieseasons/lib/screens/details.dart

```diff
@@ -324,8 +324,14 @@ class _DetailsScreenState extends State<DetailsScreen> with RestorationMixin {
                   SizedBox(height: 20),
                   CupertinoSegmentedControl<int>(
                     children: {
-                      0: Text('Facts & Info'),
-                      1: Text('Trivia'),
+                      0: Text(
+                        'Facts & Info',
+                      ),
+                      //style: CupertinoTheme.of(context).textTheme.textStyle),
+                      1: Text(
+                        'Trivia',
+                      )
+                      //style: CupertinoTheme.of(context).textTheme.textStyle),
```

Comment:
I vaguely remember `CupertinoSegmentedControl` doesn't work well with dark mode, because the UI component no longer exists on iOS 13 and above.

Reply:
Seems okay on IOS 13.1. 
Blue background with black letters for selected. Black background with blue letters for unselected.
Have not applied any styling to the its children.
Not sure what it's supposed to look like.

Reply:
On iOS 13+ `UISegmentedControl`s have a different look so I'm not sure either what it is supposed to look like. I'm okay with this since there's no actual code change (could you remove the lines that are commented out?)